src/main/java/com/careerfair/q/workflow/queue/employee/physical/implementation/PhysicalQueueWorkflowImpl.java

```diff
@@ -90,7 +91,11 @@ public EmployeeQueueData registerStudent(String employeeId, String studentId) {
         StudentQueueStatus studentQueueStatus = removeStudent(employeeId,
                 checkQueueAssociated(employee), studentId, true);
 
-        if (!studentFirebase.registerStudent(studentId, employeeId)) {
+        try {
```

Comment:
I think we should do this error handling in registerStudent itself. Error handling should be implemented as downstream as possible so that higher levels don't have to deal with them I think. What do you think?

Reply:
The only reason I am doing this is so that exceptions other than InvalidRequestExceptions are not thrown

Reply:
Oh I have a FirebaseException. Changing it to InvalidRequest

Reply:
Gotcha